### PR TITLE
[Fix #124] Move dependencies to gemspec and untrack Gemfile.lock

### LINKED ISSUE #124: Thing about development dependencies and removing Gemfile.lock
See https://github.com/datarockets/datarockets-style/pull/119#discussion_r342550405

### PR DESCRIPTION
**Pull request by paydaylight**
* Moved gem dependencies as suggested in [this](https://github.com/datarockets/datarockets-style/pull/119#discussion_r342550405) comment.
* Removed `Gemfile.lock`

Before you submit a pull request, please make sure you have to follow:

- [x] read and know items from the [Contributing Guide](https://github.com/datarockets/datarockets-style/blob/master/CONTRIBUTING.md#pull-requests)
- [x] add a description of the problem you're trying to solve (short summary from related issue)
- [ ] verified that cops are ordered by alphabet
- [ ] add a note to the style guide docs (if it needs)
- [x] add a note to the changelog file
- [x] the commit message contains the number of the related issue (if it presents)
  and word `Fix` if this PR closes related issue
- [x] squash all commits before submitting to review

### CODE REVIEW

File: datarockets-style.gemspec [hunk 1]
```diff
@@ -30,6 +30,9 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_dependency "pry-byebug"
+  spec.add_dependency "rake", "~> 13.0"
+  spec.add_dependency "rspec", "~> 3.10"
```
Comment (roman-dubrovsky) [inline datarockets-style.gemspec:35]: it should be development dependency. I don't want to have it as a dependency in the project

**Review by roman-dubrovsky — CHANGES_REQUESTED** [review]

File: datarockets-style.gemspec (hunk 1 above)
Comment (paydaylight) [reply datarockets-style.gemspec]: @Roma fixed

Comment (paydaylight) [reply datarockets-style.gemspec]: @roman-dubrovsky fixed

File: lib/datarockets_style/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module DatarocketsStyle
-  VERSION = "1.1.0".freeze
+  VERSION = "1.0.0".freeze
```
Comment (roman-dubrovsky) [inline lib/datarockets_style/version.rb:2]: we've already released `1.1.0` so we should not downgrade the version

**Review by roman-dubrovsky — CHANGES_REQUESTED** [review]

File: lib/datarockets_style/version.rb (hunk 2 above)
Comment (paydaylight) [reply lib/datarockets_style/version.rb]: @roman-dubrovsky my bad, fixed

**Review by roman-dubrovsky — APPROVED** [review]

**Pull request closed without merging**